src/routes/NoPermissionsPage/NoPermissionsPage.tsx

```diff
@@ -1,10 +1,9 @@
 import React, { useEffect } from 'react';
-import { withRouter } from 'react-router-dom';
 
 import { Main } from '@redhat-cloud-services/frontend-components/Main';
 import { NotAuthorized } from '@redhat-cloud-services/frontend-components/NotAuthorized';
 
-function NoPermissionsPage() {
+const NoPermissionsPage = () => {
```

Comment:
nit: Some of components are still defined with `function` keyword, but this and `OopsPage.tsx` use arrow function definition.

While rare, there are some weird edge cases that can occur with top-level arrow functions, so I'd prefer we leave them with the `function` keyword.